debug: true
map_frame: map
odom_frame: odom
base_frame: base_footprint

robot_radius: 0.27

# sample
dock:
  dock_speed: 0.1
  dock_distance: 0.2
  pose_x: 1.334
  pose_y: 4.134
  pose_z: 0.0
  orientation_x: 0.0
  orientation_y: 0.0
  orientation_z: -0.691
  orientation_w: 0.723

